docs/starters.yml

```diff
@@ -6301,6 +6301,19 @@
     - Sitemap Generation
     - XML Sitemaps
     - Progressive Web Ap
+- url: https://gatsby-basic-typescript-starter.netlify.app/
+  repo: https://github.com/noahub/gatsby-typescript-starter
+  description: This starter ships with the main Gatsby configuration files you need to build a basic site using React and Typescript.
+  tags:
```

Comment:
Can you refer to our whitelist of tags (https://github.com/gatsbyjs/gatsby/blob/master/docs/categories.yml) under the `starters` key to find alternative tags to use here? 🙂 Looks like Language:Typescript, Styling:Css-in-JS might be good ones.

Reply:
@gillkyle thanks for taking a look. I've added your suggested tags.